README.md

```diff
@@ -79,15 +79,15 @@ Note that all instance methods throw an error when something goes wrong or when
 
 **SMTPClient.prototype.connect({timeout})**:Promise;
 
-> Connects to the SMTP server and starts socket I/O activity.
+> Connects to the SMTP server and starts to socket I/O activity.
 
 | Option | Type | Required | Default | Description
 |--------|------|----------|---------|------------
 | timeout | Integer | No | 0 | A time in milliseconds after the operation automatically rejects (`0` disables the timeout).
 
 **SMTPClient.prototype.data(source, {sourceSize, timeout})**:Promise;
 
-> Sends the DATA command to the server which uploads the `source` of an email and finalize the process with the `.` (automatically appended to the source).
+> Sends the DATA command to the server which uploads the `source` of an email and finalizes the process with the `.` (automatically appended to the source).
```

Comment:
Finalize can in this instance be interpreted as command, therefore it is correct.